activities/tool-management/plausible-analytics.md

```diff
@@ -0,0 +1,21 @@
+---
+type: resource
+---
+
+# Plausible Analytics
+
+Plausible Analytics is a simple, lightweight (< 1 KB), open-source and privacy-friendly alternative to Google Analytics. It doesn’t use cookies and is fully compliant with GDPR, CCPA and PECR. The Plausible team are dedicated to making web analytics more privacy-friendly. Their mission is to reduce corporate surveillance by providing an alternative web analytics tool which doesn’t come from the AdTech world.
+
+## Using Plausible Analytics
+
+Plausible Analytics is integrated into our Jekyll-themed website built with GitHub Pages. With Plausible activated, we have access to a simple open source analytics dashboard. This dashboard includes website metrics such as the number of unique visitors, the number of page views on individual pages and the referral sources which have sent traffic to our site. We will use this data to assess the performance of our site and to figure out what we can do to improve our future efforts.
```

Comment:
```suggestion
Plausible Analytics is integrated into our [Jekyll-theme](https://github.com/publiccodenet/jekyll-theme), which is used by all our websites (since all are built with Github Pages). It gives us a simple open source analytics dashboard, including website metrics such as the number of unique visitors, the number of page views on individual pages and the referral sources that have sent traffic to our site. 

We will use this data to assess the performance of our site and to figure out what we can do to improve our future efforts.
```